packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.js

```diff
@@ -43,16 +43,18 @@ export default class QuanticCategoryFacetValue extends LightningElement {
   }
 
   get categoryFacetLiClass() {
-    if(this.activeParent) {
-      return "slds-var-m-left_large slds-grid";
-    }
-    return "slds-grid";
+    return this.activeParent ? 'slds-var-m-left_large slds-grid' : 'slds-grid';
   }
+
+  get facetValue() {
+    return this.item.value;
+  }
+  
   /**
-   * @param {InputEvent} evt
+   * @param {Event} evt
    */
   onSelect(evt) {
     evt.preventDefault();
-    this.dispatchEvent(new CustomEvent('selectvalue', {detail: this.item}));
+    this.dispatchEvent(new CustomEvent('selectvalue', {detail: (this.facetValue)}));
```

Comment:
I'd suggest to continue sending an object through `detail` because it would give us the opportunity to send more information if needed in the future in a non-breaking way.

But, as mentioned in the LWC documentation, we must create a brand new object instance when setting the event detail.

    this.dispatchEvent(new CustomEvent('selectvalue', {
      detail: {
        value: this.facetValue
      }
    }));

What do you think?

Reply:
I'll try it out